cnf-certification-test/lifecycle/suite.go

```diff
@@ -604,3 +631,12 @@ func testStorageRequiredPods(env *provider.TestEnvironment) {
 	}
 	testhelper.AddTestResultLog("Non-compliant", nonCompliantPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func checkHpa(hpaList []*scalingv1.HorizontalPodAutoscaler, name, kind string) (*scalingv1.HorizontalPodAutoscaler, bool) {
```

Comment:
This could probably be moved into a file `hpa.go` or similar to keep the `suite.go` clean.  Also helps with adding unit tests.